libnmstate/nm/bond.py

```diff
@@ -17,24 +17,59 @@
 # along with this program. If not, see <https://www.gnu.org/licenses/>.
 #
 
-from . import connection
+import os
+import re
+
 from . import nmclient
 from libnmstate.error import NmstateValueError
 
 
 BOND_TYPE = "bond"
 
+SYSFS_BOND_FOLDER = "/sys/class/net/{}/bonding"
+
+NM_SUPPORTED_BOND_OPTIONS = (
+    "mode",
```

Comment:
Please sort the tuple to make it easier to identify duplicates in case we need to check for new properties in the future.

Reply:
Sorted. Thanks.